codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/Instantiator.kt

```diff
@@ -92,6 +93,8 @@ open class Instantiator(
         fun doesSetterTakeInOption(memberShape: MemberShape): Boolean
     }
 
+    fun generate(shape: Shape, data: Node, ctx: Ctx = Ctx()) = writable { render(this, shape, data, ctx) }
```

Comment:
Given that neither this method nor `render` reference their class, I'm wondering why they're even methods. Should this functionality be moved elsewhere?

Reply:
they do eventually (e.g. they need symbolProvider)